ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1438,7 +1438,7 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
                            disabled={!fp.includes(SpecificPopulationEnum.OTHER, populationDetails)}
                            data-test-id='other-specialPopulation-text'
                            onChange={v => this.setState(fp.set(
-                             ['workspace', 'researchPurpose', 'otherPopulationDetails'], v))}/>
+                             ['workspace', 'researchPurpose', 'otherPopulationDetails'], v.trim()))}/>
```

Comment:
This is a good change, but it actually works already without v.trim().  Looks like this is due to L1037+
```
        constraints['otherPopulationDetails'] = {
          presence: {
            allowEmpty: false,
            message: tooShort
          },
```